ixofoundation/ixo-webclient
# Remove redundant Actions from Action Panel

## Scope
[Ticket: WEB-272](https://ixo.atlassian.net/browse/WEB-272)

## Work Done

- Remove buttons from Actions Panel for:
    Rate this Project
    Offer Oracle Services
    Ask for Help
    Delegate (replaced by Stake)
    Redelegate (replaced by Stake)
    Withdraw Delegation Reward
    Get Claims List (the download data will be moved into the dashboard)

- Add Stake Action button to invoke the Staking Modal
- Fuel this project change label to Add Credit 
- Conditional display of Actions
    The following Governance Actions only display in the context of an entity that has one or more Claim Template/s of Type Proposal
    Submit Proposal, Pay Deposit
    Stake Action only display in the context of Entity Type Cell and Cell Type Validator
    Apply to Join change label to Apply and only display in the context of Entity Status= Recruiting 
    Update StatusAction only display in the context of user is the Entity Creator

## Steps to Test
- select entity on entities explorer to navigate to https://app-uat.ixo.world/project/:projectDID/overview
- check Action panel

## Screenshots
